templates/industrial/index.html

```diff
@@ -0,0 +1,695 @@
+{% extends "industrial/base_industrial.html" %}
+      <!-- TODO -->
+{% block title %}TODO find title?{% endblock %}
+{% block meta_description %}Ubuntu is widely used to reduce time to market in automotive companies, as a platform for data analytics, online services, robotics and machine learning. Infotainment, ADAS and cockpit application development on Ubuntu is faster. New real-time hypervisors enable Ubuntu VMs alongside functional safety RTOS solutions.{% endblock meta_description %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1PVliB92h4ZqexaZV9F5MIcX7aRG6yBAGH1hG7x8UZTw/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-topped">
+  <div class="row u-equal-height">
+    <div class="col-8">
+      <h1>App-centric industry 4.0</h1>
+      <!-- TODO find right class for subheading -->
+      <p class="p-heading--4">Workload consolidation and micro clouds for the factory of the future</p>
+      <!-- TODO do we need rel=”noopener noreferrer” on links? -->
+      <p>Rated ‘Excellent’ for industrial edge cloud platforms in the <a href="https://ubuntu.com/engage/pac-radar">PAC Radar analyst report</a></p>
+      <!-- TODO which contact us?-->
+      <p><a class="js-invoke-modal p-button--positive" href="/internet-of-things/contact-us?product=automotive">Get in touch</a></p>
+    </div>
+    <div class="col-4 u-vertically-center u-align--center u-hide--small">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/3d44acdd-Robotics_icon.svg",
+          alt="",
+          height="250",
+          width="248",
+          hi_def=True,
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--suru">
+  <div class="row">
+    <blockquote class="p-pull-quote">
+      <p class="p-pull-quote__quote">Building our new ctrlX AUTOMATION with Ubuntu Core has enabled us to create a software-driven industrial manufacturing solution.</p>
+      <cite class="p-pull-quote__citation">Hans-Michael Krause, Director Product Management PLC and IoT, Bosch Rexroth.</cite>
+    </blockquote>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <h2 class="col-6">Empowering industry 4.0 champions</h2>
```

Comment:
I think this would be better with a clear use of the grid for each part:
```html
<div class="row">
  <div class="col-6">
    <h2>Empowering industry 4.0 champions</h2>
  </div>
</div>
<div class="row">
  <div class="col-12">
     <ul class="p-inline-images u-no-margin--bottom u-align--left">
      ...
```